source/PcAux/R/exportedHelperFunctions.R

```diff
@@ -277,27 +277,31 @@ getImpData <- function(pcAuxData) pcAuxData$miDatasets
 calcTime <- function(pcAuxData, what) {
     time     <- pcAuxData$time[[what]]  
     eachStep <- diff(time)
-    
+    ##
     nPoints                      <- length(eachStep)
     eachStep[nPoints + 1]        <- as.vector(time[length(time)] - time["start"])
     names(eachStep)[nPoints + 1] <- "overall"
-
+    ##
     usrVars <- lapply(c("End", "usr"), function(x) grep(x, names(eachStep)))
-    
+    ##
     if(length(unlist(usrVars)) > 1) {
         eachStep["overall"] <- eachStep["overall"] - eachStep[usrVars[[1]]]
         timeSteps           <- eachStep[-usrVars[[2]]]
         timeSteps["usr"]    <- eachStep[usrVars[[1]]]
     }
     else timeSteps <- eachStep
-    
-    timeSteps
+    ##
+    return(timeSteps)
 }
 
 
 ## Write machine status to a text files 
 writeStatus <- function(pcAuxData, outName, what) {
```

Comment:
This function still isn't working, but if you change its definition to something like:

writeStatus <- function(pcAuxData, outName, what) capture.output(pcAuxData$status[[what]], file = outName)

I think it will work as desired (the issue appears to be attempting to specify the connection as an un-named argument; when doing so, no connection is opened, so nothing is written to disk). 

Please update this function (possibly according to the preceding recommendation).

Reply:
Yes, I see that now. I've added the argument name so it knows "fileName" is the connection.